src/ensembl/src/content/app/species/components/species-main-view/SpeciesMainViewStats.tsx

```diff
@@ -0,0 +1,217 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { useEffect, useState, useRef } from 'react';
+import { connect } from 'react-redux';
+import classNames from 'classnames';
+
+import PointerBox, {
+  Position
+} from 'src/shared/components/pointer-box/PointerBox';
+import SpeciesStats from 'src/content/app/species/components/species-stats/SpeciesStats';
+import ExpandableSection from 'src/shared/components/expandable-section/ExpandableSection';
+
+import {
+  getActiveGenomeId,
+  getActiveGenomeStats
+} from 'src/content/app/species/state/general/speciesGeneralSelectors';
+import { getGenomeExampleFocusObjects } from 'src/shared/state/genome/genomeSelectors';
+import { fetchStatsForActiveGenome } from 'src/content/app/species/state/general/speciesGeneralSlice';
+
+import { RootState } from 'src/store';
+import { GenomeStats } from '../../state/general/speciesGeneralSlice';
+import ViewInApp, { urlObj } from 'src/shared/components/view-in-app/ViewInApp';
+import {
+  StatsSection,
+  sectionGroupsMap
+} from '../../state/general/speciesGeneralHelper';
+import { ExampleFocusObject } from 'src/shared/state/genome/genomeTypes';
+
+import styles from './SpeciesMainView.scss';
+
+type Props = {
+  activeGenomeId: string | null;
+  genomeStats: GenomeStats | undefined;
+  exampleFocusObjects: ExampleFocusObject[];
+  fetchStatsForActiveGenome: () => void;
+};
+
+const ExampleLinks = (props: {
+  exampleLinks?: Partial<urlObj>;
+  exampleLinkText?: string;
+}) => {
+  const { exampleLinks, exampleLinkText } = props;
+
+  const [showPointerBox, setShowPointerBox] = useState(false);
+  const anchorRef = useRef<HTMLSpanElement>(null);
+  const containerRef = useRef<HTMLSpanElement>(null);
+
+  if (!exampleLinks) {
+    return null;
+  }
+
+  return (
+    <div className={styles.exampleLink}>
+      <span ref={containerRef}>
+        <span
+          className={styles.exampleLinkText}
+          ref={anchorRef}
+          onClick={() => setShowPointerBox(!showPointerBox)}
+        >
+          {exampleLinkText}
+          {showPointerBox && (
+            <PointerBox
+              anchor={anchorRef.current as HTMLDivElement}
+              onOutsideClick={() => setShowPointerBox(false)}
+              position={Position.BOTTOM_RIGHT}
+              container={containerRef.current}
```

Comment:
The point of the container prop is to make the pointer box try to avoid extending beyond the borders of a given outer element:

![image](https://user-images.githubusercontent.com/6834224/95140737-7a4fad00-0767-11eb-84e1-3db8d43c7f89.png)

It defaults to `window` and is useless here.

Reply:
Yes, that's right.

Any idea how the following can be prevented? Should it be closed as soon as the user starts to scroll?
![Kapture 2020-10-06 at 14 21 59](https://user-images.githubusercontent.com/3085815/95207335-aa3a9700-07df-11eb-8776-6e4598050823.gif)

Reply:
> Should it be closed as soon as the user starts to scroll?

That's what the tooltips do. I'd say yes.

One other option is to try the `renderInsideAnchor` prop; then the pointer box will be placed into the "Example gene" span rather than at the top level of the window. This might cause other layout problems — or it may work out fine :-)